src/test/scala/com/databricks/spark/csv/CsvSuite.scala

```diff
@@ -547,6 +548,24 @@ abstract class AbstractCsvSuite extends FunSuite with BeforeAndAfterAll {
     assert(carsCopy.collect.map(_.toString).toSet == cars.collect.map(_.toString).toSet)
   }
 
+  test("DSL checking non null escapeChar is set before NONE quoteMode") {
+    // Create temp directory
+    TestUtils.deleteRecursively(new File(tempEmptyDir))
+    new File(tempEmptyDir).mkdirs()
+    val copyFilePath = tempEmptyDir + "cars-copy.csv"
+
+    val cars = sqlContext.csvFile(carsFile, parserLib = parserLib)
+    val delimiter = ","
+    var quote = "\""
+    var escape = "\\"
+    var quoteMode = "NONE"
+
+    noException should be thrownBy {
```

Comment:
I know those things such as `should be` are pretty much used in Scala as some kind of convention. However, would this be better if we have some test codes having the same format?

I have seen such things used in various open sources but not for Spark yet.